### DISCUSSION TITLE
post/8266-switch

### DISCUSSION THREAD

**Discussion by giscus** [Interstellar750/hexo_s#28] [General]
# post/8266-switch/

可接入 HomeKit 来控制开关

https://trle5.xyz/post/8266-switch/

**Comment by Interstellar750** [comment]
看了一下 Vercel 的统计数据，似乎 esp8266loader.apk 这个文件被下载了十多次，是有很多人照着这个教程搞继电器了吗 🤔
![Screenshot 2023-01-01 03 22 27](https://user-images.githubusercontent.com/38517305/210153814-c0bd26f9-bd90-4f28-81cc-4de00af7c8d6.png)
